### PR TITLE
KEYCLOAK-19442 TLS to Postgre documentation

### PR DESCRIPTION
**Pull request by jonathanvila**
[KEYCLOAK-19442](https://issues.redhat.com/browse/KEYCLOAK-19442)

Added documentation info related to the TLS connection to the Database

### CODE REVIEW

File: server_installation/topics/operator/external-database.adoc [hunk 1]
```diff
@@ -19,6 +19,7 @@ stringData:
     # Required for AWS Backup functionality
     POSTGRES_SUPERUSER: "true"
     POSTGRES_USERNAME: <Database Username>
+    SSLMODE: ""
```
Comment (vmuzikar) [inline server_installation/topics/operator/external-database.adoc]:
```suggestion
    SSLMODE: <TLS configuration for the Database connection>
```

File: server_installation/topics/operator/external-database.adoc [hunk 2]
```diff
@@ -36,6 +37,21 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+* `SSL_MODE` - Indicates, whether to use TLS on the connection to the external PostgreSQL database, check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]
+
+When `SSL_MODE` is enabled, the operator will search for a secret called `keycloak-db-ssl-cert-secret` containing the `root.crt` that has been used by the PostgreSQL database. Here is an example :
```
Comment (vmuzikar) [inline server_installation/topics/operator/external-database.adoc]:
```suggestion
When `SSL_MODE` is enabled, the operator will search for a secret called `keycloak-db-ssl-cert-secret` containing the `root.crt` that has been used by the PostgreSQL database. Creating the secret is optional and the secret is used only when you want to verify the Database's certificate (for example `SSLMODE: verify-ca`). Here is an example :
```

**Review by vmuzikar — CHANGES_REQUESTED** [review]

**Review by vmuzikar — DISMISSED** [review]
@jonathanvila LGTM, thank you!

File: server_installation/topics/operator/external-database.adoc [hunk 3]
```diff
@@ -36,6 +37,21 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+* `SSL_MODE` - Indicates, whether to use TLS on the connection to the external PostgreSQL database, check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]
```
Comment (andymunro) [inline server_installation/topics/operator/external-database.adoc:40]: Remove the comma after "Indicates"
Also, what is the subject of the clause "check the possible..." Is that a second action of this property?

**Review by andymunro — CHANGES_REQUESTED** [review]
A couple of changes, @vmuzikar

File: server_installation/topics/operator/external-database.adoc (hunk 3 above)
Comment (jonathanvila) [reply server_installation/topics/operator/external-database.adoc:40]: probably could make it clearer : 

`Indicates whether to use TLS on the connection to the external PostgreSQL database. Please check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]`

Comment (jonathanvila) [reply server_installation/topics/operator/external-database.adoc:40]: Checking it again, the previous text for POSTGRES_SUPERUSER is using a comma after the Indicates, should I remove it too ? @andymunro

**Comment by jonathanvila** [comment]
[Moved to KEYCLOAK-19442](https://github.com/keycloak/keycloak-documentation/pull/1309)

**Pull request closed without merging**